wazo_dird/database/queries/personal.py

```diff
@@ -86,15 +86,19 @@ def _new_user_contacts_filter(self, user_uuid):
 
 
 class PersonalContactCRUD(BaseDAO):
-    def list_personal_contacts(self, user_uuid=None):
+    def list_personal_contacts(self, user_uuid=None, search_params={}):
```

Comment:
one should avoid mutable default parameter values, as the literal `{}` is only evaluated once on import, and the same reference to the created `dict` object gets reused for any subsequent call to the method(hence any mutation on the value will affect all subsequent call with that same default value).
https://docs.python-guide.org/writing/gotchas/
```suggestion
    def list_personal_contacts(self, user_uuid=None, search_params=None):
        search_params = {} if search_params is None else search_params
```